src/block/utils/ComposterUtils.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\block\utils;
+
+use pocketmine\block\VanillaBlocks;
+use pocketmine\item\Apple;
+use pocketmine\item\BakedPotato;
+use pocketmine\item\BeetrootSeeds;
+use pocketmine\item\Bread;
+use pocketmine\item\Carrot;
+use pocketmine\item\CocoaBeans;
+use pocketmine\item\Cookie;
+use pocketmine\item\DriedKelp;
+use pocketmine\item\Item;
+use pocketmine\item\ItemBlock;
+use pocketmine\item\ItemIdentifier;
+use pocketmine\item\ItemIds;
+use pocketmine\item\Melon;
+use pocketmine\item\MelonSeeds;
+use pocketmine\item\Potato;
+use pocketmine\item\PumpkinSeeds;
+use pocketmine\item\SweetBerries;
+use pocketmine\item\WheatSeeds;
+use function count;
+
+class ComposterUtils {
```

Comment:
`ComposterUtils` is too vague. The name should be more specific to what the class actually does.

In addition, this should use `SingletonTrait` instead of being all static.